### DISCUSSION TITLE
Issue with migration - RPC call timeout

### DISCUSSION THREAD

**Discussion by kinguk** [WalletConnect/walletconnect-monorepo#2896] [Developer Support] [answered]
Hi, 

I followed the migration documentation for Web3Modal and all testing went well. It was working up until this morning.

ow for both Mainnet and Goerli I am getting timeouts for these requests:
`POST https://rpc.walletconnect.com/v1/?chainId=eip155:5&projectId=xxx`

It still works for Sepolia. Does the v1 in the url suggest the migration didnt work?

These are the packages I upgraded to:
```
    "@wagmi/core": "^1.2.0",
    "@web3modal/ethereum": "^2.2.0",
    "@web3modal/html": "^2.2.0",
    "viem": "^1.0.7",
```

**Comment by Valentin529** [comment]
Having the same issue. Clearly it began after wallet connect has shut down their v1. Apparently  either wagmi or viem is still sending request to v1 as stated in failed https query. Any ideas when it is going to be fixed?

**Comment by kinguk** [comment, marked as answer]
Looks like the RPC server is down which would be causing the outage.
https://status.walletconnect.com/

<img width="902" alt="Screenshot 2023-06-30 at 09 30 05" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/18704647/4d927292-7553-4913-be65-730d481a5eff">
